jenkins/jenkins2-ha-agents.yaml

```diff
@@ -1192,7 +1192,7 @@ Resources:
                   end
                 end
                 def awaitFile(file)
-                  endTime = Time.now.to_i + $conf['maxWaitInSeconds']
+                  endTime = Time.now.to_i + 1800
```

Comment:
Why did you decide to hardcode this, @michaelwittig?

Reply:
It's related to the startup of jekins. it has nothing to do with the user defined maxWaitInSeconds